File: 240813/11720. 숫자의 합/숫자의 합.cc
#include <iostream>
using namespace std;

int main() {
   string str;
   int n;
   int sum = 0;

   cin >> n >> str;
   for(int i = 0; i < n; i++){
        sum += (int)str[i] - 48;
   }
   cout << sum;

    
}

File: 240813/11720. 숫자의 합/숫자의 합.java
import java.util.*;

public class Main{
	
	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);
		
		int n = sc.nextInt();
		sc.nextLine();
		
		StringBuffer sb = new StringBuffer(sc.nextLine());
		
		int sum = 0;
		
		for(int i = 0; i < n; i++) {
			sum += sb.charAt(i) - '0';
		}
		
		System.out.println(sum);
		sc.close();
	}
}